File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyChemtechDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyChemtechDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyChemtechDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyChemtechDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyChemtechDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyChemtechDragonKillAnimation} from '../../../src/hud/animations/EnemyChemtechDragonKillAnimation.js';

/**
 * Integration test for enemy chemtech dragon kill animation on actual hardware.
 */
describe('EnemyChemtechDragonKillAnimation Integration', () => {
    test('playEnemyChemtechDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyChemtechDragonKillAnimation());
    }, 30000);
});
